pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -214,6 +214,10 @@ func (opt *Options) Run(args []string) error {
 		return strings.Contains(t.name, "[Smoke]")
 	})
 
+	water, normal := splitTests(normal, func(t *testCase) bool {
+		return strings.Contains(t.name, "[Water]")
```

Comment:
Water is probably too obscure (even smoke was pushing it).  Probably better to use [Pre] and [Post] or [Early] and [Late]

Reply:
Yes yes, sorry hence the WIP, just wanted an easier way to find the job :D 

Can also change the Smoke to Pre and Change to Post, sgty?

Reply:
@smarterclayton Done, PTAL, thanks!